infra/recipes/terraform/azure/modules/vm/main.tf

```diff
@@ -0,0 +1,63 @@
+// define the virtual network
+resource "azurerm_virtual_network" "virtual-network" {
+  name                = var.virtual_network_name
+  address_space       = ["10.0.0.0/16"]
+  location            = var.resource_group_location
+  resource_group_name = var.resource_group_name
+}
+
+// define the subnet, the specific area in the virtual network
+resource "azurerm_subnet" "subnet" {
+  name = var.subnet_name
+  resource_group_name =  var.resource_group_name
+  virtual_network_name = azurerm_virtual_network.virtual-network.name
+  address_prefixes = ["10.0.1.0/24"]
```

Comment:
Does it make sense to move both `address_space` and address_prefixes` to variable?

Reply:
Yes